index.html

```diff
@@ -41,6 +41,111 @@ <h3 class="connect"><a href="#">LET'S CONNECT</a></h3>
 				</ul>
 			</div>
 		</section>
+		<section class="project-works">
+			<ul class="work">
+				<li>
+					<img src="Assets/images/Portfolio2.png" alt="tonic project">
+					<h2>Tonic</h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back-end dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+			<ul class="big-project">
+				<li>
+					<img src="Assets/images/Portfolio4.png" alt="multipost-stories">
+					<h2>Multi-Post<span>Stories</span></h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back End Dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
```

Comment:
- [ ] Here the semantic tags are ok, but you should make a styling for it so that it looks a clickable element. Also follow the original design.